cmd/options.go

```diff
@@ -97,3 +97,11 @@ func WithLazyRefresh() Option {
 		c.conf.LazyRefresh = true
 	}
 }
+
+// WithConnRefuseNotify configures the Proxy to start a goroutine and run the
+// given notify callback function in the event of a connection refuse.
```

Comment:
Minor wording nit:

WithConnRefuseNotify configures the Proxy to call the provided function when a connection is refused. The notification function is run in a goroutine.

Reply:
Revised, thanks!